docs/instance_groups.md

```diff
@@ -320,3 +320,19 @@ spec:
     httpPutResponseHopLimit: 1
     httpTokens: required
 ```
+
+## maxInstanceLifetime (AWS Only)
+
+{{ kops_feature_table(kops_added_default='1.25') }}
+
+The maximum instance lifetime specifies the maximum amount of time (in seconds) that an instance can be in service before it is terminated and replaced.
+A common use case might be a requirement to replace your instances on a schedule because of internal security policies or external compliance controls. 
+In other words, this feature helps you to put a bit of ephemerality in your cluster.
+You must specify a value of at least 86,400 seconds (one day).
+
+The following configuration enables a maximum instance lifetime to 172,800 seconds (two days).
+
+```yaml
+spec:
+  maxInstanceLifetime: 87000
```

Comment:
Maybe use duration as example? I also believe that this needs quoting otherwise.

Reply:
Fixed in 43f0e2f

Reply:
Fixed in 43f0e2f